projects/frontend/data-pipelines/gui/e2e/integration/router/router.spec.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021-2023 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/// <reference types="cypress" />
+
+import { BasePagePO } from '../../support/pages/base/base-page.po';
+
+describe('Routing for pages', () => {
+    describe('smoke', { tags: ['@smoke'] }, () => {
+        it('navigates to get-started page when explore page route is ignored', () => {
```

Comment:
I figured we didn't need it since we have a spec for the whole explore page. It's enabled by default.

https://github.com/vmware/versatile-data-kit/blob/person/mdilan/dynamic-routes-app-init/projects/frontend/data-pipelines/gui/e2e/integration/explore/data-jobs/data-jobs.spec.js